docs/nGraph_DG/nGraph_Python_API.md

```diff
@@ -0,0 +1,218 @@
+
+```python
+partial_shape = node_input.get_partial_shape()
+if partial_shape.is_dynamic:
+    print('{} node has a dynamic shape {}'.format(node.get_friendly_name(), partial_shape))
+else:
+    shape = list(partial_shape.to_shape())
+    print('{} node has a static shape {}'.format(node.get_friendly_name(), shape))
+```
+
+### Node attributes
+
+Nodes may also have additional attributes. For example, a `Softmax` node has an `axis` attribute. Each attribute may be retrieved using a dedicated method, for example, `node.get_axis()`.
+
+```python
+>>> node = ng.softmax(data=[[1, 2], [3, 4]], axis=1)
+>>> node.get_axis()
+1
+```
+
+You can also set attribute values using corresponding setter methods, for example:
+
+```python
+node.set_axis(0)
+```
+
+Currently, you can get all attributes of a node using the `_get_attributes` method. Please note that this is an internal API method and may change in future versions of OpenVINO.
+
+The following code displays all attributes for all nodes in a function:
+
+```python
+for node in function.get_ordered_ops():
+    attributes = node._get_attributes()
+    if(attributes):
+        print('Operation {} of type {} has attributes:'.format(node.get_friendly_name(), node.get_type_name()))
+        for attr, value in attributes.items():
+            print("  * {}: {}".format(attr, value))
+    else:
+        print('Operation {} of type {} has no attributes.'.format(node.get_friendly_name(), node.get_type_name()))
+```
+
+### Node runtime information
+
+Attributes are properties of nodes in the computational graph, which will be stored when the model is serialized to a file. However, there can be additional properties of nodes, which are only important during the execution of a graph. An example of such a property is `affinity`, which determines which operation will be executed on which hardware in a heterogeneous environment. You can get and set runtime information by using the `get_rt_info` method of a node.
+
+```python
+rt_info = node.get_rt_info()
+rt_info["affinity"] = "test_affinity"
+```
+
+Please note that the `rt_info` API may change in future versions of OpenVINO.
+
+### Data of Constant nodes
+
+You can retrieve the data tensor associated with a `Constant` node by calling its `get_data` method:
+
+```python
+if node.get_type_name() == 'Constant':
+    data = node.get_data()
+``` 
+
+## Optimize graphs
+
+nGraph supports a growing number of optimization passes, which can be applied to a graph to increase model execution performance. To perform constant folding of subgraphs, you can use the `ConstantFolding` pass. To do this, create an optimization pass `Manager`, register the passes you need to execute, and feed an nGraph function to the configured pass manager.
```

Comment:
We do not suggest user to call any transformations to increase model performance neither in C++ nor in Python API. All available transformations required usually for some advanced cases.
So I suggest to rename this section to `Apply transformations` and replace `nGraph supports a growing number of optimization passes, which can be applied to a graph to increase model execution performance.` with smth like `nGraph provides some transformations that could be applied to ngraph::Function and usually used for  internal needs.` And also I would add `Please note that the list of available transformations may change in future versions of OpenVINO.`